src/main/java/redis/clients/jedis/JedisPubSub.java

```diff
@@ -15,11 +15,9 @@
 import redis.clients.jedis.exceptions.JedisException;
 import redis.clients.util.SafeEncoder;
 
-public abstract class JedisPubSub {
+public abstract class JedisPubSub extends BinaryJedisPubSub {
```

Comment:
@marcosnils  I didn't get you what exactly you mean by "subclass and downmerge without subclassing" ? Can you please help me understanding that.

Also whenever you and @HeartSaVioR  decides the final way to go then please let me know , i will make the changes accordingly under both of you guidance.